README.md

```diff
@@ -1,24 +1,190 @@
+  </ul>
+</details>
+<details>
+  <summary>Visual Studio Code</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com">Visual Studio Code</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/html5/">HTML5</a></li>
+    <li><a href="https://www.css3.com/">CSS</a></li>
+  </ul>
+</details>
+ 
+## Key Features
+
+- User Authentication
+- React Redux 
+- ROR API endpoints
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Example command:
+
+```sh
+1.use a browser
+2.use cable internet
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+
+  git clone git@github.com:emhamza/my_new_rails_app.git
+  cd my_new_rails_app
+
+
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+bundle install
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+- Open the terminal to play with the commands.
+```
+
+### Run tests
+
+To run tests for the linter, run the following command:
+
+Example command:
+
+```sh
+rspec
+bundle exec rspec spec
+```
+
+```sh
+rubocop -A
+```
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Umair Hamza**
+
+- GitHub: [@emhamza](https://github.com/emhamza)
+- Twitter: [@uhamzaz](https://twitter.com/uhamzaz)
+- LinkedIn: [Umair Hamza](https://www.linkedin.com/in/umair-hamza-a8262b261/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- As an ambitious project, we have a roadmap for future enhancements. This includes incorporating advanced features such as recipe favoriting, meal planning, and social sharing. We also plan to implement additional testing, including performance testing and automated UI testing.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [Isuues Pages](https://github.com/emhamza/ror-api-backend/issues)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly give it a star⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the immense support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/emhamza/ror-api-backend/blob/dev/LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<a name="readme-top"></a>
```

Comment:
- [ ] the link to the MIT license is not working as expected please make sure it is referring to the actual license.

Reply:
Certainly, thank you for pointing that out 🎉